www/js/diary.js

```diff
@@ -3,9 +3,8 @@ import LabelTab from './diary/LabelTab';
 
 angular.module('emission.main.diary',['emission.main.diary.services',
                                       'emission.survey.multilabel.buttons',
-                                      'emission.survey.multilabel.infscrollfilters',
                                       'emission.survey.enketo.add-note-button',
-                                      'emission.survey.enketo.trip.infscrollfilters',
```

Comment:
I saw the comment related to this, but I also wonder where we are using `emission.main.diary` any more. It used to be in `main.js` for the routing, but we have also switched to react routing now, correct? can't we just remove this file?